assignments/objects.js

```diff
@@ -18,27 +18,73 @@ const example = {
 }
 
 // Write your intern objects here:
+const mitzi = {
+	"id": 1,
+	"name": "Mitzi",
+	"email": "mmelloy0@psu.edu",
+	"gender": "F"
+}
+
+const kennan = {
+	"id": "2",
+	"name": "Kennan",
+	"email": "kdiben1@tinypic.com",
+	"gender": "M",
+	speak: function() {
+		return "Hello, my name is Kennan!";
+	}
+}
+
+const keven = {
+	"id": "3",
+	"name": "Keven",
+	"email": "kmummery2@wikimedia.org",
+	"gender": "M"
+}
+
+const gannie = {
+	"id": "4",
+	"name": "Gannie",
+	"email": "gmartinson3@illinois.edu",
+	"gender": "M"
+}
+
+const antonietta = {
+	"id": "5",
+	"name": "Antonietta",
+	"email": "adaine5@samsung.com",
+	"gender": "F",
+	multiplyNums: function(a, b) {
+		return (a*b);
+	}
+}
+
```

Comment:
Awesome job putting the interns' associated action functions into their objects!